src/diffusers/models/attention.py

```diff
@@ -184,6 +184,10 @@ class BasicTransformerBlock(nn.Module):
         attention_head_dim (`int`): The number of channels in each head.
         dropout (`float`, *optional*, defaults to 0.0): The dropout probability to use.
         cross_attention_dim (`int`, *optional*): The size of the encoder_hidden_states vector for cross attention.
+        only_cross_attention (`bool`, *optional*):
+            Whether to use only cross-attention layers. In this case two cross attention layers are used.
+        double_self_attention (`bool`, *optional*):
+            Whether to use two self-attention layers. In this case no cross attention layers are used.
```

Comment:
```suggestion
        only_cross_attention (`bool`, *optional*):
            Whether to use only cross-attention layers. In this case two cross attention layers are used.
        only_self_attention (`bool`, *optional*):
            Whether to use only self-attention layers. In this case two self attention layers are used.
```

Reply:
Problem here is that `only_self_attention` is the default case when no cross dim is provided. Also, it'd be difficult to convince people that `only_self_attention` means 2 self attention layers I tihnk.

Reply:
Ah makes sense